addOns/help_ru_RU/src/main/javahelp/contents/ui/dialogs/options/globalexcludeurl.html

```diff
@@ -3,34 +3,34 @@
 <head>
 <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
 <title>
-Options Global Exclude URL screen
+Опции Global Exclue URL-экран 
```

Comment:
There's a typo in exclude.

Reply:
So should we close this PR?

Reply:
Fine to merge to me, not that a big issue and next update will take care of it.